### ISSUE TITLE
Filtering models visibility by clicking

### ISSUE THREAD

**Issue by JoshuaJMoore** [Geodan/i3dm.export#18]
Hi, is it possible to filter the visibility of one of the models on the tile when clicking on it?

**Comment by bertt** [comment]
Do you mean clicking one instance (for instance a tree) and change visibility? What's the usecase? Anyway, this should be handled in some custom client code, see https://github.com/geodan/mapbox-3dtiles . It's closely related to getting attribute information when hovering over an instance.

**Comment by JoshuaJMoore** [comment]
Thanks for your reply. Yes, exactly. In our system, when we click on an instance, we want to hide the tile version of that instance, and show a local version instead so the user can move it by dragging it, or rotate it by using a rotation handle. 

Currently in our system, we only have local instances using three box because we never invested the time into 3D tiles. From what you've said, it seems like it wont be too difficult to implement this. I did see the Query rendered features function, I just wasn't sure if the functionality for changing the visibility of an instance was already there somewhere before I looked into this more.
